Missing fs-extra in devDependencies

Got this error with fresh `npm install`:

```
module.js:471
    throw err;
    ^

Error: Cannot find module 'fs-extra'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/wp-e2e-webdriver/lib/helper.js:28:16)
    at Module._compile (module.js:570:32)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/wp-e2e-webdriver/lib/index.js:8:15)
    at Module._compile (module.js:570:32)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/gedex/Works/Automattic/wc-e2e-page-objects/test/cart-page.js:8:1)
    at Module._compile (module.js:570:32)
    at loader (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at /Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/mocha/lib/mocha.js:222:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/mocha/lib/mocha.js:219:14)
    at Mocha.run (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/mocha/lib/mocha.js:487:10)
    at Object.<anonymous> (/Users/gedex/Works/Automattic/wc-e2e-page-objects/node_modules/mocha/bin/_mocha:459:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
npm ERR! Test failed.  See above for more details
```

--- a/package.json
+++ b/package.json
@@ -43,7 +43,6 @@
     "wp-e2e-webdriver": "0.11.0"
   },
   "devDependencies": {
-    "config": "^1.24.0",
     "babel": "^6.5.2",
     "babel-cli": "^6.14.0",
     "babel-eslint": "^7.0.0",
@@ -52,6 +51,8 @@
     "babel-preset-stage-2": "^6.13.0",
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
+    "chromedriver": "^2.25.0",
+    "config": "^1.24.0",
     "cross-env": "^3.0.0",
     "eslint": "^3.11.1",
     "eslint-config-airbnb": "^13.0.0",
@@ -60,11 +61,11 @@
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
     "eslint-plugin-wpcalypso": "^3.0.2",
+    "fs-extra": "^2.1.2",
     "istanbul": "^1.0.0-alpha",
     "jsdoc": "^3.4.3",
     "minami": "^1.1.1",
     "mocha": "^3.0.2",
-    "rimraf": "^2.5.4",
-    "chromedriver": "^2.25.0"
+    "rimraf": "^2.5.4"
   }
 }

